openupgradelib/openupgrade.py

```diff
@@ -2486,6 +2487,30 @@ def update_module_moved_fields(
     )
 
 
+def safe_unlink(records, do_raise=False):
+    """Allow for errors to occur during unlinking of records.
+
+    Prevent broken database transactions, and by default, catch exceptions.
+
+    :param records: an iterable (not necessarily recordset) of records to
+    unlink.
+    :param do_raise: when set to True, don't catch the exception but only log
```

Comment:
It should be "When set to False" don't you think ?

Reply:
No, that's the default value. But when set to True, it does raise.

Reply:
Yes.

So better to replace ``when set to True, don't catch the exception but only log  an error`` by  ``when set to True, don't catch the exception and raise the error``.
Do'nt you think ?

Reply:
Docstring updated.